models.go

```diff
@@ -44,6 +44,16 @@ func (c Checklist) Completed() bool {
 	return true
 }
 
+// CompletedChecksOfUser returns whether all the items are checked by any user.
```

Comment:
Maybe: "CompletedChecksOfUser returns whether all the items of user are checked"?